api/scpca_portal/test/test_loader.py

```diff
@@ -0,0 +1,1737 @@
+            input_bucket_name=settings.AWS_S3_INPUT_BUCKET_NAME,
+            reload_existing=True,
+            update_s3=False,
+        )
+
+        self.generate_computed_files = partial(
+            loader.generate_computed_files,
+            max_workers=10,
+            update_s3=False,
+            clean_up_output_data=False,
+        )
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        # shutil.rmtree(common.OUTPUT_DATA_PATH, ignore_errors=True)
+
+    def purge_extra_samples(self, project: Project, sample_of_interest: str) -> None:
+        """Purges all of a project's samples that are not the sample of interest."""
+        for sample in project.samples.all():
+            if sample != sample_of_interest:
+                sample.purge()
+
+    def assertObjectProperties(self, obj: Any, expected_values: Dict[str, Any]) -> None:
+        for attribute, value in expected_values.items():
+            msg = f"The actual and expected `{attribute}` values differ in {obj}"
+            if isinstance(value, list):
+                self.assertListEqual(getattr(obj, attribute), value, msg)
+            else:
+                self.assertEqual(getattr(obj, attribute), value, msg)
+
+    def assertDictIsNonEmpty(self, d: Dict) -> None:
+        self.assertTrue(any(key for key in d))
+
+    def assertLibraries(self, project_zip: ZipFile, expected_libraries: Set[str]) -> None:
+        self.assertLibrariesMetadata(project_zip, expected_libraries)
+        self.assertLibrariesDataFiles(project_zip, expected_libraries)
+
+    def assertLibrariesMetadata(self, project_zip: ZipFile, expected_libraries: Set[str]) -> None:
+        file_list = project_zip.namelist()
+
+        # Check via metadata file
+        metadata_file_name = next(file_name for file_name in file_list if "metadata" in file_name)
+        metadata_file = project_zip.read(metadata_file_name)
+
+        with io.StringIO(metadata_file.decode("utf-8")) as metadata_file_str:
+            metadata_file_dict_reader = DictReader(metadata_file_str, delimiter="\t")
+            metadata_file_libraries = set(
+                row["scpca_library_id"] for row in metadata_file_dict_reader
+            )
+            self.assertEqual(expected_libraries, metadata_file_libraries)
+
+    def assertLibrariesDataFiles(self, project_zip: ZipFile, expected_libraries: Set[str]) -> None:
+        data_file_paths = [Path(file) for file in project_zip.namelist()]
+        data_file_libraries = set(
+            # data files have paths that look like "SCPCS999990/SCPCL999990_processed.rds"
+            file_path.name.split("_")[0]
+            for file_path in data_file_paths
+            if file_path.name.startswith("SCPCL")
+        )
+        self.assertEqual(expected_libraries, data_file_libraries)
+
+    def assertCreateProjectSucceeded(self, project):
+        msg = "create_project failed and didn't return a project"
+        self.assertIsNotNone(project, msg)
```

Comment:
No need for an extra method here, just call self.assertIsNotNone with the custom message
msg = "Unable to test <test name> because project was unable to be imported"